index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+      <link rel="stylesheet" href="master.css">
+    <meta charset="utf-8">
+    <title></title>
```

Comment:
The title here is what you see next to the icon on top of the tab in which the website is opened.